packages/salesforcedx-utils-vscode/src/helpers/validations.ts

```diff
@@ -28,6 +28,14 @@ export function isAlphaNumString(value: string | undefined): boolean {
   return value !== undefined && value !== '' && !/\W/.test(value);
 }
 
+export function isRecordId(value: string = '', prefix: string): boolean {
```

Comment:
I'd rather name this `isRecordIdFormat` since a string satisfying these conditions is not necessarily a record id in the org